index.js

```diff
@@ -105,6 +105,8 @@ var genericValidationRules = {
 
 function validateGenericEvent (event) {
   assert(type(event) === 'object', 'You must pass a message object.')
+  assert((JSON.stringify(event)).length < 32 << 10, 'Your message must be < 32kb.')
```

Comment:
Is this the best way to do this? Internally we use `Buffer` (https://github.com/segmentio/api-validator/blob/master/lib/rules.js#L174-L184), though I'm not sure why.

Reply:
I have no idea why we use a `Buffer` there.

Reply:
So i dug in, and it's because Strings are variable byte encoded, which means `str.length != new Buffer(str).length`. Updated to use Buffer.byteLength